config/core/deployments/webhook.yaml

```diff
@@ -42,12 +42,12 @@ spec:
         resources:
           requests:
             # taken from serving.
-            cpu: 20m
-            memory: 20Mi
+            cpu: 100m
+            memory: 100Mi
```

Comment:
That's a really high default value for a webhook, and depends a lot on the activity within the cluster. 1000 namespaces is already a high tier, for a very specific production use-case. If every component starts requesting high-tier production values by default, my next local `ko apply` is gonna be fun. I believe we should stick to a sane, lower-tier expectation by default, just enough to let people deploy Knative and play with it under casual load. 20Mi is enough for this.

I agree with your comment that this should be configurable via the operator

Reply:
yeah, I see that.  but IMO `20Mi` than is also not really something that works well for a bigger sized prod env.

Reply:
Not related to these values at all, but overall, I think out of the box you should be able to run some workloads, I think we should have a doc that describes what's supported (rough numbers), and then have 'tuning guides'. Rather than anticipate the highest loads, let's document what we support out of the box and help user configure the setup that fits their needs.